docs/technical-details/reference/resource-instance-binding.md

```diff
@@ -0,0 +1,711 @@
+# Resource Class with Resource Instance Binding behavior
+
+This document describes the legacy behavior of the Resource class using the instance binding behavior. It is recommended that [updated behavior](./resource.md) instead, but this legacy API is preserved for backwards compatibility.
```

Comment:
The second sentence reads weird. Should it be like "It is recommended to update..." or something like that?